### ISSUE TITLE
fine-tune this model in tensorflow  with 'nan' problem 

### ISSUE THREAD

**Issue by lisztfrancis** [yahoo/open_nsfw#29]
I've converted this model to tensorflow framework,  and lauch the graph with .npy data file, the data file upload without any problem since I printed all value in tf.all_variables() for debug . But I found it cann't backprop properly,the trainable variables became nan at the first bp process. I'm not very skillful at CNN tricks since my physics background.  Do I need adjust some special layers of this net? and what's the proper 
method of optimization? 
Thanks for any insightful and helpful analysis and advice!

**Comment by loretoparisi** [comment]
@lisztfrancis I was thinking the same thing, at the end I preferred Caffe. The performances on the api using Tornando Web Server + Docker is super with this ResNet-50 pre-trained model. 
Which advantage did you find converting to TF?
